validation-test/Reflection/reflect_Optional_Any.swift

```diff
@@ -8,6 +8,7 @@
 
 // REQUIRES: reflection_test_support
 // REQUIRES: executable_test
+// REQUIRES: OS=macosx || OS=tvos || OS=watchos || OS=ios
```

Comment:
```suggestion
// REQUIRES: VENDOR=apple
```

Reply:
VENDOR? I'm not at my computer. What does that even mean for Swift's test suite? In Apple's history, they've vended various non-Darwin OSes, so that suggestion feels horribly imprecise